setup.py

```diff
@@ -70,7 +70,7 @@ def run_tests(self):
         'alembic',
         'argparse',
         'jsonpath-rw>=1.2.1',
-        'openpyxl<2.1.0',
+        'openpyxl==2.5.12',
```

Comment:
> this makes it substantially easier to run commcare hq and commcare-export in the same python environment (and generally seems good to upgrade libraries).

Using `==` will make this much harder as we then need to upgrade both of these libraries. 

as an aside: I think its a bad workflow to use the same virtualenv anyways unless we explicitly want to support that and write tests for it or include commcare-export as a dependency in commcare-hq. different projects with different environments is what virtualenv is made for.

Reply:
> Using == will make this much harder as we then need to upgrade both of these libraries.

Oops. What's the right thing to do? ~=?

>  I think its a bad workflow to use the same virtualenv anyways unless we explicitly want to support that and write tests for it or include commcare-export as a dependency in commcare-hq. different projects with different environments is what virtualenv is made for.

Totally agree. I'm working on a project that explicitly needs to run them in the same env and so I'm looking for a reasonable way to do it and this seemed like the approach that made the most sense (can ping you on that PR when it is ready if you want)

Reply:
my local machine:
```
$ ls ~/.virtualenvs/ | wc
     41      41     609
```

Reply:
> Oops. What's the right thing to do? ~=?

That's hard to say and its a library by library thing. Generally if you want libraries to work together robustly, you should make them as loose as possible. Judging from https://openpyxl.readthedocs.io/en/2.6/changes.html you may want `>=2.1,<3` whatever the minimum version that's required is.

I think the way most libraries test this out is to use `tox` to create travis builds for each version, so in this example it would test openpyxl 2.1, 2.2, 2.3, 2.4, 2.5, 2.6.

Reply:
Gotcha. Am guessing I can figure out what versions are compatible / incompatible some way and change it. Before I go down that road, I'm just trying to understand what the value proposition of that effort is and make sure it's a good use of time.

Does your comment still apply if we continue to expect people to (mostly) use `commcare-export` as a standalone tool? I'm not totally sure where the upgrade dependencies/conflicts would arise.

I guess I'm tempted to just change it to `>=` and call it a day, but maybe I'm missing something.

Reply:
Yeah. I think that's fine if this is a one-off and does not need to be maintained. Just wanted to make a note that the discrepancy between making two repository's libraries compatible and using `==`